creator-node/src/routes/files.js

```diff
@@ -117,14 +113,14 @@ const streamFromFileSystem = async (
       )
     }
 
-    // If content is premium, set cache-control to no-cache.
+    // If content is gated, set cache-control to no-cache.
```

Comment:
a callout for you - even tho you've made a new route, this `getCID` function is used for all content requests so pls be careful about potential regressions with any changes to this function